### PR TITLE
Unify validation of partitioning

### PR DESCRIPTION
**Pull request by jknphy**
Unify validation of partitioning. In order to have a more general and reusable approach to validate partition in this PR we avoid to call any function in library [lib/partitions_validator_utils](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/partitions_validator_utils.pm) from test modules in YaST group.

- Related ticket: [poo#91755](https://progress.opensuse.org/issues/91755)
- Job Group MR: [MR#351](https://gitlab.suse.de/qsf-y/qa-sle-functional-y/-/merge_requests/351)
- Verification run: [SLE](https://openqa.suse.de/tests/overview?build=jknphy%2Fos-autoinst-distri-opensuse%23unify_partition_validation&version=15-SP3&distri=sle) | [openSUSE TW](https://openqa.opensuse.org/tests/overview?distri=opensuse&build=jknphy%2Fos-autoinst-distri-opensuse%23unify_partition_validation&version=Tumbleweed) | [openSUSE Leap](https://openqa.opensuse.org/tests/latest?arch=x86_64&distri=opensuse&flavor=DVD&machine=64bit&test=autoyast_reinstall_gnome%40jknphy%2Fos-autoinst-distri-opensuse%23unify_partition_validation&version=15.3#live) | [Maintenance - QR - SLE15SP2](https://openqa.suse.de/tests/overview?distri=sle&version=15-SP2&build=jknphy%2Fos-autoinst-distri-opensuse%23unify_partition_validation)

### CODE REVIEW

File: test_data/yast/autoyast_reinstall/autoyast_reinstall_64bit.yaml [hunk 1]
```diff
@@ -4,9 +4,5 @@ paths:
   - /var/lib/polkit/.bashrc
   - /var/lib/nobody/.bashrc
   - /var/lib/pulseaudio/.bashrc
-device: vda
-table_type: gpt
-file_system:
-  /: btrfs
-swap: 1
+<<: !include test_data/yast/autoyast_reinstall/partitioning_no_home.yaml
 <<: !include test_data/yast/autoyast/profiles/reinstall.yaml
```
Comment (sofiasyria) [inline test_data/yast/autoyast_reinstall/autoyast_reinstall_64bit.yaml:8]: Maybe here?

Comment (jknphy) [reply test_data/yast/autoyast_reinstall/autoyast_reinstall_64bit.yaml:8]: I notice for autoyast mini the same, when I have two includes it doesn't work, do you know why? maybe I just missing something trivial...

Comment (jknphy) [reply test_data/yast/autoyast_reinstall/autoyast_reinstall_64bit.yaml:8]: I found the issue in the script, see [commit](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12467/commits/423bd035ac181d5c696d1d9bd6faf1b6671de608#diff-373ab693c2f2beb001873bf44403dfe9b3cb9def1c54fb774cd80049bbae6752R26)

**Comment by OleksandrOrlov** [comment]
Thank you Joaquin! The changes will improve structure of our tests.

**Pull request merged**